libraries/config/src/chain_config.cpp

```diff
@@ -52,6 +52,7 @@ decltype(ChainConfig::predefined_) const ChainConfig::predefined_([] {
     cfg.final_chain.state.genesis_balances[root_node_addr] = 9007199254740991;
     auto& dpos = cfg.final_chain.state.dpos.emplace();
     dpos.eligibility_balance_threshold = 1000000000;
+    dpos.coins_per_vote = 1000000000;
```

Comment:
isnt this supposed to be 100k ?

Reply:
It is used for tests. So to not mess values and tests it should be the same as `eligibility_balance_threshold`